sdk/tests/deqp/framework/common/tcuSkipList.js

```diff
@@ -96,10 +97,28 @@ goog.scope(function() {
     tcuSkipList.getSkipStatus = function(testName) {
         var skipEntry = _skipEntries[testName];
         if (skipEntry === undefined) {
-            return { 'skip': false, 'reason': '' };
+            return this.getSkipsStatus(testName);
         } else {
             return { 'skip': true, 'reason': skipEntry };
         }
     }
 
+    /*
+     * Gets the skip status of the given tests like testpath*
+     * object with the properties "skip", a boolean, and "reason", a
+     * string.
+    */
+    tcuSkipList.getSkipsStatus = function(testName) {
+        var skipEntry;
+        for (var key in _skipEntries) {
+            if (testName.indexOf(key) >=0 ) {
```

Comment:
It has been the case in the past that lookups of test expectations with wildcards has been a major source of slowdowns of the WebGL conformance suite. Please see http://crbug.com/522234 and https://codereview.chromium.org/1310123002 .

Sorry, but I don't think we should make this change. There shouldn't be that many entries in the skip list, and I think those tests can be added explicitly. How many do you predict will be needed?